var/spack/repos/builtin/packages/plasma/package.py

```diff
@@ -48,22 +55,35 @@ class Plasma(MakefilePackage):
     conflicts("%xl_r")
 
     patch("remove_absolute_mkl_include.patch", when="@17.1")
+    patch("add_netlib_lapacke_detection.patch", when="@17.1")
+
+    def getblaslapacklibs(self):
+        if '^netlib-lapack' in self.spec:
+            bl_attr = ':c'
+        else:
+            bl_attr = ''
+        return self.spec['lapack' + bl_attr].libs + \
```

Comment:
if you will be adjusting `libs`, don't forget to add here extra `self.spec['lapack'].libs` and `self.spec['blas'].libs`